dotcom-rendering/src/web/components/MostViewedFooterGrid.tsx

```diff
@@ -147,18 +154,23 @@ export const MostViewedFooterGrid = ({
 										: unselectedStyles,
 									isFirst && firstTab,
 								]}
-								role="tab"
-								aria-selected={isSelected}
-								aria-controls={`tabs-popular-${i}`}
+								role="none"
 								id={`tabs-popular-${i}-tab`}
 								data-cy={`tab-heading-${i}`}
-								key={`tabs-popular-${i}-tab`}
+								key={`tabs-popular-${tab.heading}-tab`}
 								data-link-name={tab.heading}
 								data-chromatic="ignore"
 							>
-								<button
+								<a
+									href={`tabs-popular-${i}`}
```

Comment:
Why no longer a `button`?

Reply:
The rationale is that it's because it's used for page navigation. It's something that we discussed with the DAC and I think the feeling was that you could legitimately do either, but `<a>` seemed the preferred option iir.

Reply:
tbh in practice it shouldn't make a difference because if the element is hydrated (which it should be, if there are tabs) then the `role=tab` should override button and link roles. Using an anchor tag is kind-of a nod towards progressive enhancement, although I don't think it would actually work without js, as things are